INITIALIZE_FROM_FILE requires a 'current_user'

Related to #50, it looks like this feature may have been broken in recent user-based changes.

Specifically:
```
[ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/workspace/app/__init__.py", line 61, in <module>
    create_from_json(Config.INITIALIZE_FROM_FILE)
  File "/workspace/app/models.py", line 374, in create_from_json
    upsert(DatasetModel, query={ "name": name}, update=dataset_json)
  File "/workspace/app/models.py", line 347, in upsert
    new_model.save()
  File "/workspace/app/models.py", line 45, in save
    self.owner = current_user.username
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'NoneType' object has no attribute 'username'
```

It looks like in `models.py:45`, 
```
self.owner = current_user.username
```
there will not be any `current_user` available when the app is initializing; maybe there should be a special 'system' owner? Or perhaps the `initializer.json` file could contain a user section which would associate the datasets/categories created with a particular user...

Or maybe this feature should just be replaced with an API call so it will always have an associated user?
Then again, that would create a need to manually create users before being able to use the API...probably at least need a builtin `admin` account for such bootstrap concerns...

--- a/app/models.py
+++ b/app/models.py
@@ -42,7 +42,10 @@ class DatasetModel(db.DynamicDocument):
             ImageModel.load_images(directory, self.id)
 
         self.directory = directory
-        self.owner = current_user.username
+        if current_user:
+            self.owner = current_user.username
+        else:
+            self.owner = 'system'
 
         return super(DatasetModel, self).save(*args, **kwargs)
 
@@ -195,7 +198,11 @@ class AnnotationModel(db.DynamicDocument):
         if self.color is None:
             self.color = color_util.random_color_hex()
 
-        self.creator = current_user.username
+        if current_user:
+            self.creator = current_user.username
+        else:
+            self.creator = 'system'
+
         return super(AnnotationModel, self).save(*args, **kwargs)
 
     def is_empty(self):
@@ -257,6 +264,8 @@ class CategoryModel(db.DynamicDocument):
 
         if current_user:
             self.creator = current_user.username
+        else:
+            self.creator = 'system'
       
         return super(CategoryModel, self).save(*args, **kwargs)
 

